src/main/java/com/wanderersoftherift/wotr/datagen/WotrModelProvider.java

```diff
@@ -96,6 +96,81 @@ protected void registerModels(BlockModelGenerators blockModels, @NotNull ItemMod
                         Variant.variant().with(VariantProperties.MODEL, baseRiftSpawnerModel))
                 .with(createFacingDispatchFromUpModel()));
 
+        blockModels.createTrivialBlock(WotrBlocks.NOGRAVGRAVEL.get(),
+                TexturedModel.CUBE.updateTexture(mapping -> mapping.put(TextureSlot.ALL,
+                        ResourceLocation.fromNamespaceAndPath("minecraft", "block/gravel"))));
```

Comment:
use ResourceLocation.withDefaultNamespace instead (only takes the location string, not the "minecraft".)